### DISCUSSION TITLE
edgeComputing/mht

### DISCUSSION THREAD

**Discussion by giscus** [LaoYutang/blogGiscus#8] [Announcements]
# edgeComputing/mht

猕猴桃是本电科技旗下的带宽资源共享软件，根据用户共享的带宽，提供共享收益。

https://blog.laoyutang.cn/edgeComputing/mht.html

**Comment by 435325553** [comment]
没搜索到镜像，删了镜像？？？？？

**Reply by LaoYutang** [reply]
镜像 tag 变了，官方搞事情，变成 docker pull lwmacct/bd-cdn:71edge-ipes-nas-t2306150 了

**Reply by LaoYutang** [reply]
或者直接不指定 tag 也行，本电老是会删掉旧镜像

**Comment by VOX-CNNO1** [comment]
少见的mht教程。不过我这边联通大内网小水管mht收益最稳，每天6-24都是顶着跑，不过只跑18-24最划算，反正没在线时长限制
